pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -629,120 +587,134 @@ In this step, a non-native Marathon instance is installed on DC/OS with the Meso
 
-    ### Disabled
-
-    -   Full access
-
-        ```bash
-        dcos:adminrouter:service:<service-name> full
-        dcos:service:marathon:<service-name>:services:/ full
-        dcos:adminrouter:ops:mesos full
-        dcos:adminrouter:ops:slave full
-        ```
+    ```bash
+    # Access to the new marathon service under `/service/<name>` URL
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:${MARATHON_INSTANCE_NAME} full
+    dcos security org users grant ${USER_ACCOUNT} dcos:service:marathon:${MARATHON_INSTANCE_NAME}:services:/ full
+
+    # Access to the Mesos tasks and their sandbox
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:mesos full
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:slave full
+
+    # (Optionally) Access to the marathon instance that runs on the root 
+    # Marathon and can be controlled via the DC/OS UI
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:marathon full
+    dcos security org users grant ${USER_ACCOUNT} dcos:service:marathon:marathon:services:/${MARATHON_INSTANCE_NAME} full
+    ```
 
-    -  Access to an individual service or group is not supported in disabled security mode.
+  - For giving permissions to an **individual** service (or group) that runs on the newly created Marathon and is named `${CHILD_SERVICE_NAME}`:
 
-    ### Permissive
+    ```bash
+    # Access to the new marathon service under `/service/<name>` URL
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:${MARATHON_INSTANCE_NAME} full
+    dcos security org users grant ${USER_ACCOUNT} dcos:service:marathon:${MARATHON_INSTANCE_NAME}:services:/${CHILD_SERVICE_NAME} full
+
+    # Access to the Mesos tasks and their sandbox
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:mesos full
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:slave full
+
+    # (Optionally) Access to the marathon instance that runs on the root 
+    # Marathon and can be controlled via the DC/OS UI
+    dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:marathon full
+    dcos security org users grant ${USER_ACCOUNT} dcos:service:marathon:marathon:services:/${MARATHON_INSTANCE_NAME} full
+    ```
 
-    -  **Full access**
+  ### Strict
 
-        ```bash
-        dcos:adminrouter:service:<service-name> full
-        dcos:service:marathon:<service-name>:services:/ full
-        dcos:adminrouter:ops:mesos full
-        dcos:adminrouter:ops:slave full
-        ```
+  For each `${USER_ACCOUNT}` that you want to give access, if you are running a cluster in `strict` security mode, depending on what kind of permissions you want to give:
 
-    -  **Access to an individual service or group**
+  - For giving **full** permissions to any service (or group) that runs on the newly created Marathon:
```

Comment:
Here too may be tabs?

Reply:
Yeah, sub-tabs are not supported at the moment. I will open a PR to fix this 👼

Reply:
Let's create a JIRA issue for this?